Python/bytecodes.c

```diff
@@ -5285,6 +5285,13 @@ dummy_func(
             value = PyStackRef_FromPyObjectBorrow(ptr);
         }
 
+        tier2 op(_SWAP_CALL_ARG_LOAD_CONST_INLINE_BORROW, (ptr/4, callable, null, arg -- res, a, c)) {
```

Comment:
Can you give this a name that describes what it does. It doesn't call anything, nor have an argument.
`callable` must be the builtin `len` function, so is immortal and can be dropped. Avoid the register shuffle:
```suggestion
        tier2 op(_LOAD_CONST_INLINE_BORROW_SWAP4_DROP, (ptr/4, callable, null, arg -- res, n, a)) {
            assert(_Py_IsImmortal(callable));
```

Reply:
Done.

Reply:
Not quite done. You don't need to shuffle the values around as much.

Reply:
If I dont shuffle them the following POP_TOP will segfault?

Reply:
It won't on the default build, as `NULL` is immortal. You'd need to check on the FT build.